### PR TITLE
GEOMESA-1406 JSON path accessors for string attributes

### PR DESCRIPTION
**Pull request by elahrvivaz**
~~note: stacks on top of #1122 ~~

### CODE REVIEW

File: build/cqs.tsv [hunk 1]
```diff
@@ -16,7 +14,7 @@ com.googlecode.juniversalchardet:juniversalchardet	1.0.3	compile
 com.google.guava:guava	14.0.1	compile
 com.google.guava:guava	17.0	compile
 com.google.guava:guava	18.0	compile
-com.jayway.jsonpath:json-path	2.0.0	compile
+com.jayway.jsonpath:json-path	2.2.0	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11927

File: build/cqs.tsv [hunk 2]
```diff
@@ -73,8 +71,8 @@ joda-time:joda-time	2.3	compile
 log4j:log4j	1.2.17	compile
 net.iharder:base64	2.3.8	compile
 net.java.dev.jsr-275:jsr-275	1.0-beta-2	compile
-net.minidev:asm	1.0.2	compile
-net.minidev:json-smart	2.1.1	compile
+net.minidev:accessors-smart	1.1	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11929

File: build/cqs.tsv [hunk 3]
```diff
@@ -73,8 +71,8 @@ joda-time:joda-time	2.3	compile
...
+net.minidev:json-smart	2.2.1	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: approved: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11928

**Comment by jnh5y** [comment]
@mdzimmerman @cfkelly can you review this one as well?

**Comment by elahrvivaz** [comment]
needs rebase - will wait until #1122 is merged to do so

**Comment by jahhulbert-ccri** [comment]
@elahrvivaz ready to rebase. i'll be out tomorrow @jnh5y if you want to merge go for it once it's rebased.

**Comment by elahrvivaz** [comment]
@jnh5y @jahhulbert-ccri rebased - ready for merge

**Pull request merged**